content/blog/eng/lambda-days-2020.md

```diff
@@ -24,7 +24,7 @@ Here is a short list of the talks that caught our attention:
 
 **John Hughes** one of Haskell creators presented his QuickCheck library and some best practises to use it.
 John is a great entertainer, and in a funny way showed what is the so called *property based testing*.
-Brefly, we could say that our code that needs testing contains properties we need to prove and to do this is good let the 
+Brefly, we could say that the code we are testing contains properties to prove and to do this is best let the 
```

Comment:
Briefly, we could say that the code we are testing contains properties that must be proven. To do this, the best practice is  to  let the testing library generate hundreds of of test cases.